app/code/Magento/Catalog/Model/Product.php

```diff
@@ -1729,6 +1729,8 @@ public function setIsDuplicable($value)
      * Check is product available for sale
      *
      * @return bool
+     * @deprecated
```

Comment:
Unfortunately we would have to deliver this change in the core repository as we cannot accept changes to the core code from Inventory repo. Please submit another PR to [`magento/magento2`](https://github.com/magento/magento2) and link it to this one.